packages/ariakit/src/composite/composite-hover.ts

```diff
@@ -71,6 +71,7 @@ export const useCompositeHover = createHook<CompositeHoverOptions>(
       // may lose some events if this component is unmounted, but others are
       // still mounted.
       addGlobalEventListener("mousemove", setMouseMoving, true);
+      addGlobalEventListener("mouseup", resetMouseMoving, true);
```

Comment:
I'd add mousedown as well just in case `MenuItem` is triggering something on mouse down.
```suggestion
      // See https://github.com/ariakit/ariakit/issues/1137
      addGlobalEventListener("mousedown", resetMouseMoving, true);
      addGlobalEventListener("mouseup", resetMouseMoving, true);
```

Reply:
Wait, I just realized this adds `mousedown` too. Does that even really matter though? My thinking was that `mouseup` would be enough